packages/usa-tooltip/src/test/tooltips.spec.js

```diff
@@ -53,6 +80,22 @@ tests.forEach(({ name, selector: containerSelector }) => {
       assert.strictEqual(tooltipBody.classList.contains("is-set"), false);
     });
 
+    it("tooltip is visible on mouseover", () => {
+      EVENTS.mouseover(tooltipTrigger);
+      assert.strictEqual(tooltipBody.classList.contains("is-set"), true);
+    });
+
+    it("tooltip is hidden on mouseout", () => {
+      EVENTS.mouseout(tooltipTrigger);
+      assert.strictEqual(tooltipBody.classList.contains("is-set"), false);
+    });
+
+    it("tooltip is hidden on escape keydown", () => {
+      tooltipTrigger.focus();
+      EVENTS.escape(body);
```

Comment:
I am struggling to get the escape event to register here. Any ideas?

Reply:
I've been playing around with this and the issue might have something to do with how we search for `activeTooltips` in the tooltip index.js file.

If I add a console log in lines `376-379`

```js
if (activeTooltips.length === 0) {
  console.log("🟥 no active tooltips");
  return;
}
```
It comes up when running my unit test

```js
  tooltips initialized at document.body
    ✔ trigger is created
    ✔ title attribute on trigger is removed
    ✔ tooltip body is created
    ✔ tooltip is visible on focus
    ✔ tooltip is hidden on blur
    ✔ tooltip is visible on mouseover
    ✔ tooltip is hidden on mouseout
🟥 no active tooltips
    1) tooltip is hidden on escape keydown
    ✔ should not allow for innerHTML of child elements 
```

Reply:
Update: Replaced `body` with `tooltipTrigger` because we know the unit test knows `tooltipTrigger` exists. Also updated the code in `index.js` to check for the `is-set` class instead of `is-visible` to  determine if there are active tooltips since it looked like the `is-visible` class wasn't being added in the test environment.